Examples/Files/third_party_vector_style.json

```diff
@@ -3,7 +3,8 @@
     "sources": {
 	"mapillary": {
 	    "type": "vector",
-	    "tiles": ["https://d25uarhxywzl1j.cloudfront.net/v0.1/{z}/{x}/{y}.mvt"]
+	    "tiles": ["https://d25uarhxywzl1j.cloudfront.net/v0.1/{z}/{x}/{y}.mvt"],
+        "attribution": "Mapillary, CC BY"
```

Comment:
Coo — it also looks like we can [use HTML to link this](https://github.com/mapbox/mapbox-gl-native/blob/fa5722479100875778d07427d817cd3c64d7aeca/platform/darwin/test/MGLAttributionInfoTests.m#L14) and the SDK will parse that into a tappable button in the ℹ️ sheet. I.e.,

```html
<a href="https://www.mapillary.com" target="_blank">© Mapillary, CC BY</a>
```

Reply:
Nit: the indentation is also off.